### ISSUE TITLE
Please avoid node dependency to "natives" module

### ISSUE THREAD

**Issue by bpasero** [DotJoshJohnson/vscode-xml#153]
Hi,

I am a developer on VSCode and wanted to reach out to inform you that this extension is either using or having a dependency to a node module "[`natives`](https://www.npmjs.com/package/natives)" that will soon no longer be supported in VSCode once we updated to node.js 8.x (as part of our Electron 2.0 update which is planned for the near term). 

The full graph is:

```
└─┬ xqlint@0.2.10
  └─┬ update-notifier@0.2.2
    └─┬ configstore@0.3.2
      └─┬ graceful-fs@3.0.11
        └── natives@1.1.0 
```

It would be great if this dependency could be avoided by either not depending on it anymore or updating the dependencies to newer versions so that this module is no longer pulled in. 

As soon as any code in your extension (including `node_modules`) calls `require("natives")` we will throw an exception to prevent this module from getting loaded.

If this is just a `devDependency` and the module is never loaded in production, then you can ignore this issue and just close it!

More details in https://github.com/Microsoft/vscode/issues/47569

Thanks!
Ben
VSCode Team

**Comment by Tyriar** [comment]
Looks like this will involve getting xqlint to update update-notifier to a more recent version.

**Comment by DotJoshJohnson** [comment]
Released in [`v2.0.0-preview.3`](https://github.com/DotJoshJohnson/vscode-xml/releases/tag/v2.0.0-preview.3).

**Comment by bpasero** [comment]
@DotJoshJohnson thanks
